package.json

```diff
@@ -104,5 +104,10 @@
     "ts-node": "10.7.0",
     "tslib": "2.3.1",
     "typescript": "4.6.3"
+  },
+  "org_entando_dependencies": {
+    "mysql": "mysql:8.0.26",
```

Comment:
put here only the version not the full image name

Reply:
done